pkgs/top-level/python-packages.nix

```diff
@@ -4998,6 +4998,18 @@ in {
 
   nanoleaf = callPackage ../development/python-modules/nanoleaf { };
 
+  affine = callPackage ../development/python-modules/affine { };
```

Comment:
Libraries in pkgs/top-level/python-packages.nix are sorted quasi-alphabetically to avoid merge conflicts.

Reply:
Can you try to sort these through the entirety of this file as alphabetical as you can?

Currently you've just alphabetically sorted the packages you've added at the very end of the file, which is not what we want.

Reply:
Sure, I'll try, but again, they sorting restarts several times throughout the file.
Let's see :)

Reply:
OK, give it another try